smartcontractkit/external-adapters-js
# BEth price feed

## Closes 22679

## Description

Implements the adapter to support the BEth price feed

## Changes

- Create a new price composite adapter.  This adapter is supposed to a generic price adapter that fetches an intermediary token price using the token allocation adapter and then calls a handler for a specific token
-  Add handler for `BEth`

## Steps to Test

1. Run price adapter
2. Send request to adapter and verify that it returns a 200
3. Try step 2 with the quote as `BTC` and `ETH`.

Sample request

```
{
  "id": "1",
  "data": {
    "from": "BETH",
    "to": "USD",
    "source": "coingecko"
  }
}
```

## Quality Assurance

- [] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
